ouroboros-consensus-cardano-test/src/Test/ThreadNet/Infra/ShelleyBasedHardFork.hs

```diff
@@ -133,14 +137,19 @@ type ShelleyBasedHardForkConstraints era1 era2 =
   , SL.TranslationError   era2 SL.ShelleyGenesis ~ Void
 
   , SL.TranslationContext era1 ~ ()
+    -- At the moment, fix the protocols together
```

Comment:
This is a fair stop-gap. Maybe not as part of this PR, but we'll quickly want a Alonzo->Babbage ThreadNet test that will necessitate fixing this.

Reply:
We'll do a follow-up PR for ThreadNet, so that we can unblock `cardano-node` development etc. Unless those tests find proper bugs, I don't think any of these changes we'll have to do for the ThreadNet tests (eg the code comment this GitHub comment is originally about) will be visible downstream.